Assets/MixedRealityToolkit-SDK/Features/Input/GazeProvider.cs

```diff
@@ -28,6 +28,10 @@ public class GazeProvider : InputSystemGlobalListener, IMixedRealityGazeProvider
 
         private const float MovementThreshold = 0.01f;
 
+        [SerializeField]
+        [Tooltip("If true, the gaze cursor will disappear when the pointer's focus is locked, to prevent the cursor from floating idly in the world.")]
+        private bool setCursorInvisibleWhenFocusLocked = true;
```

Comment:
why not just update the cursor position to the "locked" object's position?

Reply:
If this is set to false, that's what will happen. It looks a little strange, in my opinion, with the gaze cursor floating off and following the object, so I added the configuration to hide it if desired.